pkg/virt-launcher/virtwrap/network/dhcp/BUILD.bazel

```diff
@@ -5,6 +5,7 @@ go_library(
     srcs = [
         "dhcp.go",
         "ethtool.go",
+        "socket_listener.go",
```

Comment:
nit: I prefer without the `_`, but there is no consensus in general.
https://github.com/golang/go/issues/36060

Reply:
done